static/js/promotion.js

```diff
@@ -0,0 +1,34 @@
+function loadScript(url, callback) {
```

Comment:
Why not have this version of loadScript in Clojurescript so it can be available as another function arity of `load-external-js`?

```clojure
(defn load-external-js
       ( [url]
        (dom-load-one-script url))
       ([url callback]
        (dom-load-one-script-with-callback url callback)))
```